### ISSUE TITLE
[Feature request] Relative line number

### ISSUE THREAD

**Issue by lhhong** [overleaf/web#659]
As a vim user, relative line number is so important that it kind of defeats the purpose of using vim without it. I believe ace supports relative line number and hope sharelatex can integrate it.

Edit: It is only available on v1.4.x, can only be done with #652

**Comment by dirko** [comment]
To temporarily work around this I've been running 
```javascript
editor = window._debug_editors[window._debug_editors.length -1];
editor.setOption("relativeLineNumbers", "relative");
```
directly in the js console.

Maybe there is a better and more permanent way with [TamperMonkey](https://www.overleaf.com/learn/how-to/How_can_I_define_custom_Vim_macros_in_a_vimrc_file_on_Overleaf%3F)?

**Comment by jtherrmann** [comment]
I would also find this option useful, though @dirko 's solution works perfectly for now.

**Comment by MaxNicol** [comment]
Thumbs up on this issue. @dirko's solution works but is impractical since it has to be entered on each visit.

**Comment by lhhong** [comment]
Quick fix here #748

**Comment by das7pad** [comment]
Hi!

Thank you for taking the time to write up this issue.

We are in the process of migrating to a monorepo at https://github.com/overleaf/overleaf and will mark this repository read-only soon.
You can read more about the monorepo migration at https://github.com/overleaf/overleaf/issues/923.

We are going to close this issue now to avoid any confusion about the inability to comment further.

If you believe this issue still needs addressing, please create a new issue at https://github.com/overleaf/overleaf.

Thanks again!
